nixos/modules/services/networking/tailscale.nix

```
@@ -21,6 +21,12 @@ in {
       description = ''The interface name for tunnel traffic. Use "userspace-networking" (beta) to not use TUN.'';
     };
 
+    permitCertUid = mkOption {
+      type = types.str;
+      default = "";
```

using the empty string as a "not set" value isn't that great, prefer something like this:
```suggestion
      type = types.nullOr types.str;
      default = null;
```
and emit the env var only if the setting is not null

---

Fine by me.

For Tailscale it does not really change a thing https://github.com/tailscale/tailscale/blob/c591c9165379f37bf3c3d56e9105b2e24f9f99d5/ipn/ipnserver/server.go#L459-L471

---

it's more for consistency with other modules. "null means not set" is a good convention to have :)